src/provisioner/cluster.py

```diff
@@ -87,26 +87,80 @@ def _get_kvs(self, prefix, node):
             kvs.append((prefix, node))
         return kvs
 
-    def _add_component_version(self, components):
-        """Add version for each component."""
-        updated_components = []
-        for component in components:
-            component['version'] = self._cortx_release.get_version(component['name'])
-            updated_components.append(component)
-        return updated_components
+    def _get_component_version(self, component_name):
+        """Return component version."""
+        version = self._cortx_release.get_version(component_name)
+        return version
+
+    def _get_component_key_value_list(self, component_list: list, node_id: str):
```

Comment:
suggestion: _get_component_attrs() or  _get_component_kv_list()

Reply:
Renamed to _get_component_kv_list()